vendor/sigs.k8s.io/structured-merge-diff/fieldpath/path.go

```diff
@@ -35,6 +35,12 @@ func (fp Path) String() string {
 	return strings.Join(strs, "")
 }
 
+func (fp Path) Copy() Path {
```

Comment:
nit: Because the diff is small I can chase refs for looking at shallow copy handling of each element, but future generations may appreciate a comment